fbw-common/src/systems/instruments/src/EFB/Settings/Pages/SimOptionsPage.tsx

```diff
@@ -75,19 +80,22 @@ export const SimOptionsPage = () => {
             </SelectGroup>
           </SettingItem>
 
-          <SettingItem name={t('Settings.SimOptions.SyncMsfsFlightPlan')}>
-            <SelectGroup>
-              {fpSyncButtons.map((button) => (
-                <SelectItem
-                  key={button.setting}
-                  onSelect={() => setFpSync(button.setting)}
-                  selected={fpSync === button.setting}
-                >
-                  {button.name}
-                </SelectItem>
-              ))}
-            </SelectGroup>
-          </SettingItem>
+          {/* TODO: reactivate this when feature is available - deactivated in the A380X until feature is available */}
+          {!isA380X && (
```

Comment:
This is not how we should be disabling settings (by reading AC Type), see the mcduKeyboard input etc for examples on how to do this.

Reply:
See RealismPage.tsx, for EFB settings we are now using aircraftContext to inject in the flags to enable/disable certain settings. Reading airframe/AC Type is a deprecated method of determining what EFB functions to enable/disable.
```
{aircraftContext.settingsPages.realism.mcduKeyboard && (
        <SettingGroup>
          <SettingItem name={t('Settings.Realism.McduKeyboardInput')} unrealistic groupType="parent">
            <Toggle value={mcduInput} onToggle={(value) => setMcduInput(value)} />
          </SettingItem>

          {mcduInput && (
            <SettingItem name={t('Settings.Realism.McduFocusTimeout')} groupType="sub">
              <SimpleInput
                className="w-30 text-center"
                value={mcduTimeout}
                min={5}
                max={120}
                disabled={!mcduInput}
                onChange={(event) => {
                  if (!Number.isNaN(event) && parseInt(event) >= 5 && parseInt(event) <= 120) {
                    setMcduTimeout(event.trim());
                  }
                }}
              />
            </SettingItem>
          )}
        </SettingGroup>
      )}
```

Reply:
> This is not how we should be disabling settings (by reading AC Type), see the mcduKeyboard input etc for examples on how to do this.

Refactored it as per pattern.

Changed 4 files instead of one for a super simple temp hiding of an option - LOL